eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -476,8 +473,8 @@ stages:
                   - ${{if ne(artifact.skipPublishDocMs, 'true')}}:
                     - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
               WorkingDirectory: $(System.DefaultWorkingDirectory)
-              TargetDocRepoOwner: "azure-sdk"
-              TargetDocRepoName: ${{parameters.TargetDocRepoName}}
+              TargetDocRepoOwner: $(DocRepoOwner)
```

Comment:
We often use "azure-sdk" bot account for cases were we plan to create a pull request. I don't know if that is the case here but if so we might still need to bot's fork.